lib/swagger-module.ts

```diff
@@ -35,87 +40,87 @@ export class SwaggerModule {
     };
   }
 
-  public static setup(
-    path: string,
-    app: INestApplication,
-    document: OpenAPIObject,
-    options?: SwaggerCustomOptions
-  ) {
+  private static serveStatic(finalPath: string, app: INestApplication) {
     const httpAdapter = app.getHttpAdapter();
-    const globalPrefix = getGlobalPrefix(app);
-    const finalPath = validatePath(
-      options?.useGlobalPrefix && globalPrefix && !globalPrefix.match(/^(\/?)$/)
-        ? `${globalPrefix}${validatePath(path)}`
-        : path
-    );
+
     if (httpAdapter && httpAdapter.getType() === 'fastify') {
-      return this.setupFastify(
-        finalPath,
-        httpAdapter,
-        document,
-        options as FastifySwaggerCustomOptions
-      );
+      const fastifyApp = app as NestFastifyApplication; // TODO: figure out cleaner solution
```

Comment:
that's fine 👍

Reply:
I've added a suggestion in the function signature that maybe is a way to remove the need of the `as` here :)